server/steganography/sequence/sequence.control.coffee

```diff
@@ -0,0 +1,68 @@
+module.exports 'sequenceControl',
+    # prywatna funkcja, która sprawdza, czy użytkownik osiągnął poprzedni krok sekwencji
+    # oraz czy z osiąganiem kolejnego "wyrobił się" w czasie
+    _checkStep: (number, informations, updatedAtMost = 10) ->
```

Comment:
w sumie... :) wiem tylko, że w językach, gdzie w klasach nie można deklarować prywatnych metod (np. Python) to często nazywa się je zaczynając od _ i wszyscy trzymają się od nich z daleka :)